model/release.go

```diff
@@ -146,11 +147,57 @@ func (tag *Tag) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+type Notes string
+
+var (
+	NotesLengthMaximumCharacters = 1024
+
+	ErrReleaseNotesTooLong  = errors.New("release notes too long")
+	ErrCharactersNotAllowed = errors.New("release notes contain characters which are not allowed")
+)
+
+type InvalidCharError struct {
+	Offset int
+	Char   byte
+}
+
+func (err InvalidCharError) Error() string {
```

Comment:
(nitpick) Generally, it's preferred to have pointer receiver for errors. This way `errors.Is` compares by pointer reference instead of struct header.
```suggestion
func (err *InvalidCharError) Error() string {
```

Reply:
sure, no problem.